### PR TITLE
DO NOT MERGE: Data build for #693

### PR DESCRIPTION
**Pull request by matentzn**
Data build for #693

### CODE REVIEW

**Review by joeflack4 — APPROVED** [review]
LGTM!

File: src/ontology/reports/component_signature-omim.tsv
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/reports/component_signature-omim.tsv:1]: @joeflack4  Haven't you made some changes to OMIM recently? Can you think of some ways you could confirm if these gene additions and removals make sense by spot checking a few entries?

File: src/ontology/reports/sync-synonym/synonym_sync_combined_cases.robot.tsv
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/reports/sync-synonym/synonym_sync_combined_cases.robot.tsv:1]: Have you created/updated any aliases for any commands that is deleting this file?

**Review by twhetzel — APPROVED** [review]
LGTM

File: src/ontology/reports/sync-synonym/synonym_sync_combined_cases.robot.tsv
[the remark this answers is not in the document]
Comment (joeflack4) [reply src/ontology/reports/sync-synonym/synonym_sync_combined_cases.robot.tsv:1]: I haven't touched the makefile goals for this for some weeks I think.

File: src/ontology/reports/component_signature-omim.tsv
[the remark this answers is not in the document]
Comment (joeflack4) [reply src/ontology/reports/component_signature-omim.tsv:1]: From what I can see, my local `mondo-genes.robot.tsv` in `omim` repo should be a good reference to compare the HGNC IDs that appear there against what's in the mirror/component signatures in `mondo-ingest`. This file doesn't get committed so I can't check a diff, but face checking a few add/deletions, it looks consistent.

It is in a very separate part of the pipeline than the Gene-to-Disease stuff we've been working on.

The source data comes from `mim2gene.txt`, but it doesn't have HGNC IDs there. It has symbols, which get looked up on a known map to get the IDs. But I think the face checking is good enough. Still I'm in the spot of "looks like OMIM made a lot of updates recently" without knowing much else.

**Pull request closed without merging**